config/helm/chart/default/templates/Common/crd/dynatrace-operator-crd.yaml

```diff
@@ -4442,6 +4442,10 @@ spec:
                       type: object
                     type: array
                 type: object
+              monitoredEntityID:
```

Comment:
Should we rename that to KubernetesClusterMEID?

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3447/commits/3d3d6a44b98a09ef8d9842a3ea3e263480209cbe